### PR TITLE
feat: support mt5 model

### PR DESCRIPTION
**Pull request by WongGawa**
# What does this PR do?

Fixes # (issue)

Adds # (feature)
support mt5 pipeline inference.
tested on ms2.3 release version + 910*
update how to use mt5 in readme

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you read the [contributor guideline](https://github.com/mindspore-lab/mindone/blob/master/CONTRIBUTING.md)?
- [ ] Did you make sure to update the documentation with your changes? E.g. record bug fixes or new features in `What's New`. Here are the
      [documentation guidelines](https://github.com/mindspore-lab/mindcv/wiki/%E6%96%87%E6%A1%A3%E7%BC%96%E5%86%99%E8%A7%84%E8%8C%83)
- [ ] Did you build and run the code without any errors?
- [ ] Did you report the running environment (NPU type/MS version) and performance in the doc? (better record it for data loading, model inference, or training tasks)
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

@xxx

### CODE REVIEW

**Review by tomzw11 — APPROVED** [review]

File: mindone/transformers/README.md [hunk 1]
```diff
@@ -191,6 +191,30 @@ logits = outputs[0]
...
+
+tokenizer = AutoTokenizer.from_pretrained("google/mt5-small", revision="refs/pr/3")
+model = MT5EncoderModel.from_pretrained("google/mt5-small", revision="refs/pr/3")
+input_ids = tokenizer(
+     "Studies have been shown that owning a dog is good for you", return_tensors="np"
+).input_ids  # Batch size 1
+outputs = model(input_ids=Tensor(input_ids))
+last_hidden_states = outputs[0]
+```
+
```
Comment (wtomin) [inline mindone/transformers/README.md:217]: Is `google/mt5-xxl` also supported by calling a single line or the user need to manually conver the weight to safetensors format?

Comment (WongGawa) [reply mindone/transformers/README.md:217]: The users need to manually convert the weight to safetensors format.
When converting the weight by [huggingface convert](https://huggingface.co/spaces/safetensors/convert), the following error occurred.
图片

File: mindone/transformers/README.md [hunk 2]
```diff
@@ -191,6 +191,30 @@ logits = outputs[0]
 encoder_outputs = outputs[1]
 ```
 
+### mT5
+The mT5 model was presented in [mT5: A massively multilingual pre-trained text-to-text transform](https://arxiv.org/pdf/2010.11934) by Linting Xue, Noah Constant, Adam Roberts, Mihir Kale, Rami Al-Rfou, Aditya Siddhant, Aditya Barua, Colin Raffel.
+
+We have tested the following pretrained weights from huggingface hub. Any other pretrained weights of mT5 model probably also works.
+
+#### google/mt5-small
```
Comment (SamitHuang) [inline mindone/transformers/README.md:199]: please allow provide guidance on using mt5-xxl

File: mindone/transformers/README.md
Comment (geniuspatrick) [inline mindone/transformers/README.md:1]: 撤销对此文件的修改

**Review by geniuspatrick — CHANGES_REQUESTED** [review]

**Review by wtomin — APPROVED** [review]

**Review by SamitHuang — APPROVED** [review]

**Review by geniuspatrick — APPROVED** [review]
